src/js/components/Avatar/Avatar.js

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+
+import { Box } from '../Box';
+
+const Avatar = ({
+  align = 'center',
+  round = 'full',
+  size = 'xxsmall',
+  justify = 'center',
+  src,
+  height, // for warning check and discarding the value
+  width, // for warning check and discarding the value
+  ...rest
+}) => {
+  const avatarProps = {
+    align,
+    justify,
+    height: size,
+    round,
+    width: size,
+  };
+
+  if (height || width) {
```

Comment:
if we do intend to leave this then lets only warn in development,

Reply:
actually no need to make this check.browser won't have any env to know when to show or not.